python/lib/sift_py/grpc/transport_test.py

```diff
@@ -0,0 +1,170 @@
+# ruff: noqa: N802
```

Comment:
`ruff` doesn't like PascalCase for method names but that's coming from protobuf gen. This just ignores that particular rule.